qdrant-landing/content/documentation/guides/migration.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Migration to Qdrant
+weight: 180
+---
+
+# Migration
+
+Migrating data between vector databases, especially across regions, platforms, or deployment types, can be a hassle. That’s where the [Qdrant Migration Tool](https://github.com/qdrant/migration) comes in. The tool supports all your migration needs, including moving from a self-hosted instance to Qdrant Cloud and switching cloud providers. The tool also works with self-hosted OSS Qdrant. You can also use the tool to migrate data between Qdrant instances, which is especially useful when migrating from Qdrant OSS to Qdrant Cloud.
+
+You can run the migration tool on any machine where you have connectivity to both the source and the target Qdrant databases. Direct connectivity between both databases is not required. For optimal performance, you should run the tool on a machine with a fast network connection and minimum latency to both databases.
+
+In this tutorial, we will learn how to use the migration tool and walk through a practical example of migrating from other vector databases to Qdrant. 
+
+## What is the Qdrant Migration Tool?
+
+We recently released the **Qdrant Migration Tool (beta)** to make the process of migrating data between vector databases seamless. The tool:
+
+* Streams vectors in batches from a source Qdrant collection to a target Qdrant collection.  
+* Works across cloud regions, providers, or deployment types.  
+* Supports migration of vectors from other providers and other Qdrant instances into Qdrant. 
```

Comment:
```suggestion
```

Same as the `## Why use this instead of Qdrant’s Native Snapshotting?` section below.